RetailCoder.VBE/UnitTesting/AssertClass.cs

```diff
@@ -213,7 +228,198 @@ public void AreNotSame(object expected, object actual, string message = null)
                 return;
             }
 
-            AssertHandler.OnAssertFailed("AreNotSame", string.Concat("expected: ", expected.GetHashCode(), "; actual: ", actual.GetHashCode(), ". ", message));
+            AssertHandler.OnAssertFailed(string.Format(ParameterResultFormat, expected.GetHashCode(), actual.GetHashCode(), message).Trim());
+        }
+
+        public void SequenceEquals(object expected, object actual, string message = "")
+        {
+            if (!SequenceEquityParametersAreArrays(expected, actual, true))
+            {
+                return;
+            }
+            TestArraySequenceEquity((Array)expected, (Array)actual, message, true);
+        }
+
+        public void NotSequenceEquals(object expected, object actual, string message = "")
+        {
+            if (!SequenceEquityParametersAreArrays(expected, actual, false))
+            {
+                return;
+            }
+            TestArraySequenceEquity((Array)expected, (Array)actual, message, false);
+        }
+
+
+        [SuppressMessage("ReSharper", "ExplicitCallerInfoArgument")]
+        private void TestArraySequenceEquity(Array expected, Array actual, string message, bool equals, [CallerMemberName] string methodName = "")
```

Comment:
That's not a bad idea. I'll put a permissive implementation in when I re-write the tests.